.github/workflows/build.yml

```diff
@@ -286,7 +295,7 @@ jobs:
           archives: qtbase qtsvg qttools
           cache: true
       - name: Set up build environment
-        uses: ilammy/msvc-dev-cmd@cec98b9d092141f74527d0afa6feb2af698cfe89
+        uses: ilammy/msvc-dev-cmd@v1
```

Comment:
See https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions for why this is pinned to a specific commit. If you want to update this, the `v1` tag currently points to `0b201ec74fa43914dc39ae48a89fd1d8cb592756` (which is `v1.13.0`, whereas the current commit is `v1.12.1`).

Reply:
reverted it.